language/move-prover/bytecode-to-boogie/src/spec_translator.rs

```diff
@@ -467,12 +467,14 @@ impl<'a> SpecTranslator<'a> {
     }
 
     /// Translates a resource name with type actuals
-    fn translate_resource_name(
+    fn translate_resource_type(
         &mut self,
         id: &QualifiedStructIdent,
         _type_actuals: &[Type],
     ) -> (String, SignatureToken) {
-        // TODO: incorporate type actuals
+        // TODO: incorporate type actuals. For that, we need a translator from Type AST into
+        //   SignatureToken. However, currently generic resources are also not supported
```

Comment:
The current compiler should fully support generic resources. The bytecode verifier should prevent generic resources from being published at the top level via `move_to_sender<>` (and it will soon allow) this, but everything else ought to work. Let me know if you ran into something that didn't!

Reply:
I thought to have found this in the code somewhere, but not sure where. I removed the comment.